### PR TITLE
Adds more information to portfolio page

### PR DESCRIPTION
**Pull request by abasnfarah**

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -16,13 +16,15 @@
  * Adds a random greeting to the page.
  */
 function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+  const greetings = [
```
Comment (sahanarao-sps) [inline portfolio/src/main/webapp/script.js]: Consider changing these variable names and comment above to facts since they are no longer greetings.

**Review by sahanarao-sps — APPROVED** [review]
LGTM!

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (abasnfarah) [reply portfolio/src/main/webapp/script.js]: Will Do Thanks for the catch

**Review by sahanarao-sps — APPROVED** [review]
LGTM

**Pull request merged**